src/ashell/shell-state.c

```diff
@@ -578,17 +582,42 @@ int32_t ashell_check_control(const char *buf, uint32_t len)
     return RET_OK;
 }
 
+int32_t ashell_set_bootcfg(char *buf)
+{
+    if (!fs_exist(buf)) {
+        comms_println("File passed to cfg doesn't exist\n");
```

Comment:
I could leave them out and push them to the IDE instead.  Granted, the command line ashell will still fail quietly.